integration/src/test/resources/integration/homepage_to_competencypage.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Viewing the competency page
+  Scenario: Employee can click from the homepage to the competency page
+    Given as a kainos employee I want to get to the competencies per band page from the home page
```

Comment:
"Given as a ..." doesn't read well to me. I would suggest `Given I am on the home page`